server/webhook.go

```
@@ -0,0 +1,74 @@
+			Id:       &resourceId,
+			Filename: &filename,
+		}
+
+		resource, err := s.ResourceService.FindResource(resourceFind)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch resource ID: %v", resourceId)).SetInternal(err)
+		}
+
+		c.Response().Writer.WriteHeader(http.StatusOK)
+		c.Response().Writer.Header().Set("Content-Type", "application/octet-stream")
+		c.Response().Writer.Write(resource.Blob)
```

golangci-lint (1.64.8)

Error return value of `.Write` is not checked

(errcheck)